### PR TITLE
Add senf eslint config

### PR DESCRIPTION
**Pull request by ivanzusko**
1. config connect;
2. fix issues;
3. suppress some stuff that will be changed/replaced/removed;

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,16 @@
     "reselect": "^3.0.1"
   },
   "devDependencies": {
+    "@hellofresh/eslint-config-senf": "~4.0.2",
+    "eslint-config-senf": "https://github.com/hellofresh/eslint-config-senf.git",
```
Comment (nik-kor) [inline package.json]: why do you need both?

File: src/modules/Layout/Footer/Footer.js [hunk 2]
```diff
@@ -5,14 +5,12 @@ import './Footer.css';
...
-  return (
-    <footer className={b}>
-      <p>
-        <strong>Janus</strong> by <a href="http://hellofresh.com">HelloFresh</a>. The source code is licensed <a href="http://opensource.org/licenses/mit-license.php">MIT</a>.
-      </p>
-    </footer>
-  );
-};
+const Footer = () => (
+  <footer className={b}>
```
Comment (nik-kor) [inline src/modules/Layout/Footer/Footer.js]: why is formatting different in jsx?

Comment (ivanzusko) [inline src/modules/Layout/Footer/Footer.js]: this is strange... I've run `--fix` flag... and after that, no indentation errors left...

File: package.json (hunk 1 above)
Comment (ivanzusko) [inline package.json]: Just took it from here... was fighting some time to make it works

image

File: src/modules/Layout/Footer/Footer.js (hunk 2 above)
Comment (ivanzusko) [inline src/modules/Layout/Footer/Footer.js]: just took a look at files and found out that there is a **lot** of them with the same issue...
I need to fix them all

File: package.json (hunk 1 above)
Comment (nik-kor) [inline package.json]: probably don't need it ^^^^

**Pull request merged**